mix/CodeModel.h

```diff
@@ -131,6 +132,8 @@ class CodeModel: public QObject
 	bool hasContract() const;
 	/// Apply text document formatting. @todo Move this to editor module
 	void updateFormatting(QTextDocument* _document);
+	/// Get contract code by url. Contract is compiled on first access and cached
+	dev::bytes const& getStdContractCode(QString const& _url);
```

Comment:
What is the meaning of the "Std" in the name? What I can see from the implementation, this just compiles any contract in any file.